src/xdocs/property_types.xml

```diff
@@ -831,16 +831,62 @@
 
     <section name="SeverityLevel">
       <p>
-        This type represents the severity level of a check violation. The
-        valid options are:
+        This type represents the severity level of a module violation.
+        The following table describes the valid options:
       </p>
 
-      <ul>
-        <li><code>ignore</code></li>
-        <li><code>info</code></li>
-        <li><code>warning</code></li>
-        <li><code>error</code></li>
-      </ul>
+      <div class="wrapper">
+        <table>
+          <tr>
+            <td>Option</td>
+            <td>Definition</td>
+          </tr>
+
+          <tr>
+            <td><code>ignore</code></td>
+            <td>
+              If a violation occurs, modules with this severity are ignored as if they never
+              occurred.
+              Checkstyle will not display the violation and will not fail the build.
+              This severity can be used to disable a module in the configuration. However,
+              depending on the setting of <code>executeIgnoredModules</code> when executed, the
+              module may still run and if an exception occurs, the exception will turn into an
+              error and act as one.
+            </td>
+          </tr>
+
+          <tr>
+            <td><code>info</code></td>
+            <td>
+              If a violation occurs, modules with this severity are displayed as informational.
+              Checkstyle will display the violation, but will not fail the build.
+              This severity can be used for normal logging.
+            </td>
+          </tr>
+
+          <tr>
+            <td><code>warning</code></td>
+            <td>
+              This severity behaves exactly the same as <code>info</code> in Checkstyle.
+              It can be used to stress something that <b>MAY</b> cause future problems and may
```

Comment:
`used to let user bring user more attention to such violations ..`

we do not have opinion on what is problem, user define it by sevverity.

Reply:
Done.

New sentence is `It can be used to let user bring more attention to reviewers of such violations.` for grammar.